bitnami/elasticsearch/README.md

```diff
@@ -50,407 +50,431 @@ $ helm delete --purge my-release
+| `sysctlImage.pullPolicy`         | Kernel settings modifier image pull policy  | `Always`                |
+| `sysctlImage.pullSecrets`        | Kernel settings modifier image pull secrets | `[]`                    |
+| `sysctlImage.resources.limits`   | The resources limits for the container      | `{}`                    |
+| `sysctlImage.resources.requests` | The requested resources for the container   | `{}`                    |
+
+
+### VolumePermissions parameters
+
+| Name                                   | Description                                                                                                                                               | Value                   |
+| -------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------- |
+| `volumePermissions.enabled`            | Enable init container that changes volume permissions in the data directory (for cases where the default k8s `runAsUser` and `fsUser` values do not work) | `false`                 |
+| `volumePermissions.image.registry`     | Init container volume-permissions image registry                                                                                                          | `docker.io`             |
+| `volumePermissions.image.repository`   | Init container volume-permissions image name                                                                                                              | `bitnami/bitnami-shell` |
+| `volumePermissions.image.tag`          | Init container volume-permissions image tag                                                                                                               | `10-debian-10-r127`     |
+| `volumePermissions.image.pullPolicy`   | Init container volume-permissions image pull policy                                                                                                       | `Always`                |
+| `volumePermissions.image.pullSecrets`  | Init container volume-permissions image pull secrets                                                                                                      | `[]`                    |
+| `volumePermissions.resources.limits`   | The resources limits for the container                                                                                                                    | `{}`                    |
+| `volumePermissions.resources.requests` | The requested resources for the container                                                                                                                 | `{}`                    |
+| `kibana.elasticsearch.hosts`           | Array containing hostnames for the ES instances. Used to generate the URL                                                                                 | `[]`                    |
```

Comment:
Why are these parameters in this section?

Reply:
Forgot to add the Kibana Parameters section, fixed in 80c7257. Thanks 🙂